vm/src/builtins/builtinfunc.rs

```diff
@@ -140,8 +140,10 @@ impl PyBuiltinFunction {
     }
 }
 
-// PyCMethodObject in
-// https://github.com/python/cpython/blob/main/Include/cpython/methodobject.h
+// PyMethodDescrObject in
+// https://github.com/python/cpython/blob/main/Include/descrobject.h
+// and
+// https://github.com/python/cpython/blob/main/Objects/descrobject.c
```

Comment:
will this comments going to PyMethodDescrObject?

Reply:
Sorry, I didn't get your question 🙂

Reply:
I looked in them a little. RustPython `PyBuiltinMethod` looks like both PyCMethodObject (as its feature) and PyMethodDescrObject (as `method_descriptor`? actually I don't have a concrete idea. it looks like PyMethod rather than PyBuiltinMethod) at same time. is it? Or did I miss something?

I wonder how you determine this is closer to PyMethodDescrObject rather than PyCMethodObject

Reply:
Yes, I agree, it looks like both. I've updated the docs 👍